R/remove_funs.R

```diff
@@ -22,12 +22,22 @@ crop <- function ( x , char = " " ) {
 
 ### gsub for more than one pattern
 gsubAll <- function(string, old, new) {
-  stopifnot ( is.character(string), is.character(old), is.character(new), length(old) == length(new))
+  checkmate::assert_character(string)
+  lapply(list(old, new), FUN = checkmate::assert_character,unique=TRUE, any.missing = FALSE, min.len = 2)
+  stopifnot ( length(old) == length(new))
+  stopifnot (length(intersect(old, new))==0)                                    ### in einem Zwischenschritt unique IDs mit gleich vielen characters erzeugen
```

Comment:
Could these two `stopfinot()` lines be modified so their error message is more informative? I think especially the `intersect()` one is hard to understand for users.

Reply:
Yes, good idea.